File: apps/demo_in_swift/ElectionGuardDemo/ElectionGuardDemo/Models/BallotStyle.swift
//
//  BallotStyle.swift
//  ElectionGuardDemo
//
//  Created by Ed Snider on 8/21/20.
//  Copyright © 2020 Microsoft. All rights reserved.
//

struct BallotStyle: Codable {
    let objectId: String?
    let geopoliticalUnitIds: [String]?
    let partyIds: [String]?
    let imageUri: String?
    
    enum CodingKeys: String, CodingKey {
        case objectId = "object_id"
        case geopoliticalUnitIds = "geopolitical_unit_ids"
        case partyIds = "party_ids"
        case imageUri = "image_uri"
    }
}

File: apps/demo_in_xamarin/ElectionGuardCore/ElectionGuardCore/Elections/BallotStyle.cs
using System.Collections.Generic;

namespace ElectionGuardCore.Elections
{
    public class BallotStyle
    {
        public string ObjectId { get; set; }
        public List<string> GeopoliticalUnitIds { get; set; }
        public List<string> PartyIds { get; set; }
        public string ImageUri { get; set; }
    }
}
